core/tests/karma.conf.ts

```diff
@@ -138,6 +136,8 @@ module.exports = function(config) {
       mode: 'development',
       resolve: {
         modules: [
+          'core/tests/build_sources/assets',
```

Comment:
Do you actually need this? I thought this entire folder was just so that the backend tests had something to operate on.

Reply:
The folder is used for frontend tests too I think, we need this to override the `hashes.json; for tests.

Reply:
That's weird, it shouldn't be. Where is it used exactly?